libs/application/templates/parental-leave/src/fields/InReviewSteps/index.tsx

```diff
@@ -78,16 +94,32 @@ const InReviewSteps: FC<FieldBaseProps> = ({ application }) => {
 
   if (!isRequestingRights) steps.shift()
 
+  const dob = getExpectedDateOfBirth(application)
+  const dobDate = dob ? new Date(dob) : null
+
   return (
     <Box marginBottom={10}>
-      <Box display="flex" justifyContent="spaceBetween">
-        <Text>
-          {(screenState === 'steps' &&
-            formatMessage(parentalLeaveFormMessages.reviewScreen.desc)) ||
-            formatMessage(parentalLeaveFormMessages.reviewScreen.descReview)}
-        </Text>
+      <Box
+        display={['block', 'block', 'block', 'flex']}
+        justifyContent="spaceBetween"
+      >
+        {dobDate && (
+          <Text variant="h4" color="blue400">
+            {formatMessage(
+              parentalLeaveFormMessages.reviewScreen.estimatedBirthDate,
+            )}
+            <br />
+            {format(dobDate, 'dd.MM.yyyy')}
```

Comment:
[Same](https://github.com/island-is/island.is/blob/c0c5e36c3f63d4d0a16421ff55f742541862056c/libs/shared/constants/src/lib/date.ts#L3).

Reply:
Fixed here as well.